services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentServiceITest.kt

```diff
@@ -174,6 +180,26 @@ class ApplicationAttachmentServiceITest : DatabaseTest() {
         assertThat(result.scanStatus).isEqualTo(OK)
     }
 
+    @Test
+    fun `addAttachment when application pending should send also`() {
```

Comment:
The negative case should be checked as well. I.e. add verify to the previous test.
```
verify { applicationService wasNot Called }
```